Contributing.md

```diff
@@ -0,0 +1,69 @@
+# Contributing
+When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+# Pull Request Process
```

Comment:
I think we should ask the contributor to open an issue and make sure he adding link in the PR

Reply:
good idea, I will commit changes

Reply:
following @Omeramsc comment [https://github.com/redhat-beyond/Beyond-07-team-1/pull/2#discussion_r816754923](here) I think this is the expected